sdks/python/examples/priority/test_priority.py

```diff
@@ -0,0 +1,291 @@
+import asyncio
+from datetime import datetime, timedelta
+from random import choice
+from typing import AsyncGenerator, Literal
+from uuid import uuid4
+
+import pytest
+import pytest_asyncio
+from pydantic import BaseModel
+
+from examples.priority.worker import DEFAULT_PRIORITY, SLEEP_TIME, priority_workflow
+from hatchet_sdk import Hatchet, ScheduleTriggerWorkflowOptions, TriggerWorkflowOptions
+from hatchet_sdk.clients.rest.models.v1_task_status import V1TaskStatus
+
+Priority = Literal["low", "medium", "high", "default"]
+
+
+class RunPriorityStartedAt(BaseModel):
+    priority: Priority
+    started_at: datetime
+    finished_at: datetime
+
+
+def priority_to_int(priority: Priority) -> int:
+    match priority:
+        case "high":
+            return 3
+        case "medium":
+            return 2
+        case "low":
+            return 1
+        case "default":
+            return DEFAULT_PRIORITY
+        case _:
+            raise ValueError(f"Invalid priority: {priority}")
+
+
+@pytest.mark.asyncio()
+async def test_priority(hatchet: Hatchet) -> None:
+    test_run_id = str(uuid4())
+    choices: list[Priority] = ["low", "medium", "high", "default"]
+
+    run_refs = await priority_workflow.aio_run_many_no_wait(
+        [
+            priority_workflow.create_bulk_run_item(
+                options=TriggerWorkflowOptions(
+                    priority=(priority_to_int(priority := choice(choices))),
+                    additional_metadata={
+                        "priority": priority,
```

Comment:
this confused me for a second because i thought we set priority through the additional meta prop... which maybe is a good idea...

Reply:
made it a lot easier to eyeball if this was working